public/widget-list.json

```
@@ -1,184 +1,449 @@
 [
   {
-    "name": "Carte",
-    "url": "https://infrastructure.pages.gitlab.donnees.incubateur.anct.gouv.fr/grist-widgets/carte/index.html",
-    "widgetId": "@anct/widget-map#map",
+    "name": "data.gouv.fr",
+    "url": "https://grist-plugin.opendatateam.fr",
+    "widgetId": "@dinum/data-gouv",
     "published": true,
-    "accessLevel": "read table"
+    "accessLevel": "full",
+    "renderAfterReady": true,
+    "description": "Permet à grist d'intéragir avec les données de data.gouv.fr. Nécessite votre clé API data.gouv.",
```

```suggestion
    "description": "Permet à grist d'interagir avec les données de data.gouv.fr. Nécessite votre clé API data.gouv.",
```

Pas d'accent à interagir en fait :)